target/linux/ath79/files/drivers/net/ethernet/atheros/ag71xx/ag71xx_main.c

```diff
@@ -581,6 +581,7 @@ static void ath79_mii0_ctrl_set_if(struct ag71xx *ag)
 		mii_if = AR71XX_MII0_CTRL_IF_GMII;
 		break;
 	case PHY_INTERFACE_MODE_RGMII:
+	case PHY_INTERFACE_MODE_RGMII_ID:
```

Comment:
Please make this changes separate commit.

Reply:
Such a separate commit to ag71xx could apply to both ar71xx and ath79, allowing for the ar71xx edition of this PR to be re-added on top of the ath79 edition.